dbusmock/templates/bluez5.py

```diff
@@ -112,12 +130,43 @@ def AddAdapter(self, device_name, system_name):
         ('Register', 'ss', '', ''),
         ('Unregister', 's', '', ''),
     ])
+    adapter.AddMethods(LEDADVERTISING_MNGR_IFACE, [
+        ('RegisterAdvertisement', 'oa{sv}', '', ''),
+        ('UnregisterAdvertisement', 'o', '', '')
+    ])
```

Comment:
Is this and the following method used by anything in `bluetoothctl`? Can you please add at least some cursory test to tests/test_bluez5.py to make sure that the API here is what the official tools expect?